### ISSUE TITLE
Compile darknet with setup.ps1 with opencv and gpu

### ISSUE THREAD

**Issue by Goru1890** [AlexeyAB/darknet#7393]
I just compiled darknet with setup.ps1 but it is compiled without gpu and opencv.

Is there a way to compile darknet with setup.ps1 using gpu and opencv?

@cenit

**Comment by cenit** [comment]
can you please provide logs from powershell?

**Comment by Goru1890** [comment]
![imagen](https://user-images.githubusercontent.com/35274844/108349088-ded54b00-71e2-11eb-8ad8-03a7301543e6.png)

**Comment by Goru1890** [comment]
![imagen](https://user-images.githubusercontent.com/35274844/108349218-02989100-71e3-11eb-945c-3981eb8fa206.png)

**Comment by Goru1890** [comment]
Is this enough information for you?
![imagen](https://user-images.githubusercontent.com/35274844/108350235-4a6be800-71e4-11eb-859b-d24e8d9179b7.png)

**Comment by Goru1890** [comment]
I can also see in the log, that it doesn't recognize choco-exe.

**Comment by cenit** [comment]
you're right. Setup scripts are really not very well tested and it might be that the second use is already "broken". Let me fix it

**Comment by Goru1890** [comment]
I'm getting now this error with this setup.ps1, hope it helps you.

![imagen](https://user-images.githubusercontent.com/35274844/108483101-9bd6ae80-729a-11eb-9f22-4f0dc53e61af.png)

[setup.txt](https://github.com/AlexeyAB/darknet/files/6008778/setup.txt)

**Comment by Goru1890** [comment]
> it might be that the second use is already "broken"

Can it be "fixed" with versioning? https://devblogs.microsoft.com/cppblog/take-control-of-your-vcpkg-dependencies-with-versioning-support/

**Comment by cenit** [comment]
yes, it might be, just to avoid falling inside a bug which might slip in an untested (by us) version published on vcpkg.
But opencv4 is very well tested and it should work.
Can you please send me also the install-x64-windows-dbg-out.log which is described in the error above?

**Comment by Goru1890** [comment]
The problem was that my workstation ran out of space, now I tried it again and it worked.
